packages/deployments/contracts/contracts/core/connext/facets/RoutersFacet.sol

```diff
@@ -501,9 +477,10 @@ contract RoutersFacet is BaseConnextFacet {
     address payable _to,
     address _router
   ) external nonReentrant whenNotPaused {
-    // Caller must be the router owner
-    if (msg.sender != getRouterOwner(_router)) revert RoutersFacet__removeRouterLiquidityFor_notOwner();
-
+    // Caller must be the router owner, if defined, else the router
```

Comment:
this is misleading: judging by the `initializeRouter` method, the owner will always be defined as the `msg.sender` in that fn or w/e was passed in.

If `owner == address(0)` here then the router is uninitialized: I think we could revert here in that case?

Reply:
The `owner` can be set to `0` via `proposeRouterOwner()` and `acceptProposedRouterOwner()`.
(not sure if this is desired behavior)

Reply:
that is desired behavior, and you should still *at least* be able to withdraw liquidity to the recipient (which should revert if owner is 0) using the EOA

the hard requirement being the ability to change the recipient can be removed